src/persist-client/src/lib.rs

```diff
@@ -309,15 +309,21 @@ impl PersistClient {
                 Arc::clone(&self.metrics),
             )
         });
-        let reader_id = ReaderId::new();
-        let (shard_upper, read_cap) = machine.register(&reader_id).await;
+
+        let (reader_id, writer_id) = (ReaderId::new(), WriterId::new());
+        // TODO: these registrations can be combined into a single state transition
```

Comment:
I've been making an effort to keep the number of unactionable TODOs down to a minimum within src/persist-client. I think it's fine to just drop this one, but also feel free to reword to be something like "If it ever matters, we could add a state command to do both of them at once."

(I actually wonder if the real answer is to delete `open` entirely to encourage people to only get readers and writers that they're not going to immediately expire/drop, but that should definitively be its own PR)

(edit: in the interim should we just replace the implementation of open with calling both open_reader and open_writer in sequence?)